src/app/shared/issue-pr-card/issue-pr-card-reviewers/issue-pr-card-reviewers.component.css

```diff
@@ -0,0 +1,18 @@
+.reviewer-avatar {
+  border-radius: 50%;
+  border: 2px solid white;
+  transition: all 0.3s ease-in-out;
+}
+
+.collapsed {
+  margin-left: -17px;
+}
+
+.expanded {
+  margin-left: 0px;
+}
+
+.reviewers {
+  display: flex;
+  flex-direction: row-reverse;
+}
```

Comment:
consider using css to target avatar margins on hover instead of handling this via javascript.
```css
.reviewer-avatar {
/* ... */
  margin-left: -17px; /* collapsed on default */
}

/* ... */
.reviewers:hover > .reviewer-avatar {
  margin-left: 0px;
}
```